src/User/resources/views/mail/twofactorcode.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the 2amigos/yii2-usuario project.
+ *
+ * (c) 2amigOS! <http://2amigos.us/>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+use yii\helpers\Html;
+
+/**
+ * @var String  $code
+ * @var \Da\User\Model\Token $token
+ */
+?>
+<p style="font-family: 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif; font-size: 14px; line-height: 1.6; font-weight: normal; margin: 0 0 10px; padding: 0;">
+    <?= Yii::t('usuario', 'Hello') ?>,
+</p>
+<p style="font-family: 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif; font-size: 14px; line-height: 1.6; font-weight: normal; margin: 0 0 10px; padding: 0;">
```

Comment:
The string *This is the code to insert to enable two factor authentication* is missing in the HTML email